c/ldv-linux-3.4-simple/43_1a_cilled_true-unreach-call_ok_nondet_linux-43_1a-drivers--platform--x86--topstar-laptop.ko-ldv_main0_sequence_infinite_withcheck_stateful.cil.out.c

```diff
@@ -3659,6 +3662,24 @@ void ldv_blast_assert(void)
   ERROR: __VERIFIER_error();
 }
 }
+
+void* ldv_successfull_alloc(size_t size)
```

Comment:
A typo: sucessful**L** (And the original name was `ldv_successful_malloc` not `ldv_successful_alloc`, but it doesn't matter IMHO).